lib/trento.pm

```diff
@@ -0,0 +1,300 @@
+# SUSE's openQA tests
+#
+# Copyright 2017-2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Functions for Trento tests
+
+## no critic (RequireFilenameMatchesPackage);
+package trento;
+
+use strict;
+use warnings;
+use testapi;
+use mmapi 'get_current_job_id';
+use utils qw(script_retry);
+use Carp qw(croak);
+use Exporter 'import';
+
+our @EXPORT = qw(
+  TRENTO_AZ_PREFIX
+  get_resource_group
+  get_vm_name
+  get_acr_name
+  az_get_vm_ip
+  VM_USER
+  SSH_KEY
+  cypress_install_container
+  PODMAN_PULL_LOG
+  CYPRESS_LOG_DIR
+  cypress_exec
+  cypress_test_exec
+  cypress_log_upload
+);
+
+=head1 SYNOPSIS
+Package with common methods and default values for Trento tests
+
+=cut
+
+use constant TRENTO_AZ_PREFIX => 'openqa-trento';
+# Parameter 'registry_name' must conform to the following pattern: '^[a-zA-Z0-9]*$'.
```

Comment:
I don't understand this comment. :thinking:

Reply:
prefix is like **openqa-trento** with a dash in the middle. Most of the names are composed from this one. It is readable thanks to the dash.
Unfortunately the Azure ACR service does not support dash or underscore in the names, so only letter and number, that result in a less readable string like the one for **TRENTO_AZ_ACR_PREFIX**. The comment is a justification about the not so readable name. It is due to an Azure requirement.